assets/xline_conf.tera

```diff
@@ -0,0 +1,17 @@
+# Required
+
+[cluster]
+name = "{{ name }}"
+is_leader = {{ is_leader }}
+
+[cluster.members]{% for name, addr in members %}
```

Comment:
Should we split this into two lines?

Reply:
I'm afraid it might not work out. Splitting it into two lines would yield the following outcome:
```toml
[cluster.members]

name-1="addr-1"
```

Reply:
Using the command-line approach to pass parameters to xline might be more favorable, as it eliminates the need for mounting additional configuration volumes and facilitates easier formatting.
I will remove these in the subsequent pull request.